src/rules/selector-attribute-quotes/__tests__/index.js

```diff
@@ -0,0 +1,145 @@
+import { testRule } from "../../../testUtils"
+import rules from "../../../rules"
+import { ruleName, messages } from ".."
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: ["always"],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "[title] { }",
+  }, {
+    code: "a[target=\"_blank\"] { }",
+  }, {
+    code: "a[ target=\"_blank\" ] { }",
+  }, {
+    code: "[class|=\"top\"] { }",
+  }, /* {
+    code: "[class |= \"top\"] { }",
```

Comment:
Why? I much prefer not to have unexplained mysteries like this in the code. The resolution can be after the merge, but let's make a note of what's going on now.